antenna/app.py

```diff
@@ -19,7 +20,54 @@
 from antenna.util import de_null
 
 
-logger = logging.getLogger('gunicorn.error')
+logger = logging.getLogger(__name__)
+
+
+_logging_initialized = False
+
+
+def setup_logging(config):
+    """Initializes Python logging configuration
+
+    NOTE(willkg): This causes some problems since it'll get initialized using
+    the first configuration it was given. Pretty sure that'll only happen when
+    running tests and we're not testing logging in tests. If that ever changes,
+    we'll need to revisit this.
+
+    """
+    global _logging_initialized
+    if _logging_initialized:
+        return
+
+    dc = {
+        'version': 1,
+        'disable_existing_loggers': True,
+        'formatters': {
+            'development': {
+                'format': '%(levelname)s %(asctime)s %(name)s %(message)s',
+            },
+        },
+        'handlers': {
+            'console': {
+                'level': 'DEBUG',
+                'class': 'logging.StreamHandler',
+                'formatter': 'development',
+            },
+        },
+        'root': {
+            'handlers': ['console'],
+            'level': 'WARNING',
+        },
+        'loggers': {
+            'antenna': {
+                'propagate': False,
+                'handlers': ['console'],
+                'level': 'DEBUG',
+            },
+        },
+    }
```

Comment:
I lifted this from Normandy and did some minor changes. When Antenna becomes ops-ified, we'll likely change how logging is configured and how it works. Given that, I figured I'd do a "good enough first pass that I can get stuff done" pass and defer any more thought until later.

This has logging messages getting to gunicorn and being spit out to the console.